README.md

```diff
@@ -40,84 +39,160 @@ Contact mikeb@cc.gatech.edu for more information on the Aggie project.
   `openssl req -x509 -newkey rsa:2048 -keyout key.pem -out cert.pem -days 365`
     - This will allow you to start the server but it will generate unsafe warnings in the browser. You will need a real trusted certificate for production use.
     - Adding the `-nodes` flag will generate an unencrypted private key, allowing you to run tests without passphrase prompt
 1. Run `npm install` from the project directory.
     - This installs all dependencies and concatenates the angular application.
-    - On MacOS Mojave v10.14, this command fixed most errors: `env LDFLAGS="-mmacosx-version-min=10.9" CXXFLAGS="-mmacosx-version-min=10.9" npm install`
-1. (optional) Run `npm install -g gulp mocha karma-cli protractor`.
-    - This installs gulp, mocha, karma, and protractor globally so they can be run from the command line for testing. You will most likely need Google Chrome installed on your computer for the protractor tests to run.
-    - This is optional, as `npm` provides easy access to the local copies of these that are installed by `npm install`
-1. (optional) Run `npm install -g migrate`.
-    - This installs node-migrate globally.
-1. To start server in production, run `npm start`. Use `npm run dev` for development.
+1. (optional) Run `npm install -g gulp mocha karma-cli protractor migrate`.
+    - This installs some tools globally which can then be run from the command line for testing.
+    - You will most likely need Google Chrome installed on your computer for the protractor tests to run.
+    - This is optional, as `npx` provides easy access to the local copies of these that are installed by `npm install`
+1. To start server in production mode, run `npm start`. Use `npm run dev` for development.
     - In your terminal, a user and password were generated. You will use these credentials to log into the application. Example: `"admin" user created with password "password"`.
 1. Navigate to `https://localhost:3000` in your browser.
     - This will show you the running site. Login with the user name and password from your terminal mentioned above.
     - If you did not set up the SSL certificate, use `http://localhost:3000` instead
 
+### Semi-automated installation script
+
+This is intended for setup on a fresh Ubuntu v18.04 system. Setup may need to be modified for other linux systems.
+
+If it says "user input", you won't want to paste anything beyond that until addressing the input.
+
+```shell script
+# Set up system
+
+export EDITOR=vim # Option 1
+export EDITOR=nano # Option 2
+
+sudo apt update
+sudo apt install -y ntp nginx software-properties-common
+sudo systemctl enable ntp
+sudo add-apt-repository universe
+sudo add-apt-repository ppa:certbot/certbot
+sudo apt update
+sudo apt install -y certbot python3-certbot-nginx
+
+sudo curl -o /etc/nginx/sites-available/aggie.conf https://raw.githubusercontent.com/TID-Lab/aggie/develop/docs/content/aggie-nginx
```

Comment:
Note: Until this PR is merged, use `docker-removal` instead of `develop` here.

Reply:
Looks like I got to excited :sweat_smile:, I'll swap in the right config file when I copy-paste the setup script over and run it :+1: